### PR TITLE
Bugfix/FOUR-10201: Enhanced headers and colors for copilot

### PR DESCRIPTION
**Pull request by agustinbusso**
## Issue & Reproduction Steps
Describe the issue this ticket solves and describe how to reproduce the issue (please attach any fixtures used to reproduce the issue).

Steps to reproduce:
- Login as admin
- Go to Designer 
- Click on Scripts
- Click on +Script
- Set the configuration of the script
- Click on Cornea AI Assistant
- Click on Generate Script from Text
- Set a Description
- Click on Generate

Current behavior:
The application needs the following changes:

Adjust and enhance the header of the co-pilot panels to ensure consistency.
Enhance the colors of the co-pilot to achieve greater harmony.
Change the typography of the tittles

Expected behavior:
The colors and typography should harmonize 

## Solution
- Changed Apply button title to Apply changes
- Fixed Padding left and right of AI Generated Response
- Added border to headers
- Fix font to fit the right panel (removed bold)
- Fit height to fit the right panel

**Screenshot**
Screenshot 2023-08-30 at 10 57 22

## How to Test
- Follow steps above

## Related Tickets & Packages
- [FOUR-10201](https://processmaker.atlassian.net/browse/FOUR-10201)

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

[FOUR-10201]: https://processmaker.atlassian.net/browse/FOUR-10201?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

ci:next
ci:deploy

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-5859060fdd.eng.processmaker.net

File: resources/js/processes/scripts/components/ScriptEditor.vue [hunk 1]
```diff
@@ -438,6 +439,23 @@ export default {
       window.ProcessMaker.EventBus.$emit("new-changes");
       this.handleAutosave();
     },
+    // showDiffEditor() {
```
Comment (TinCodes) [inline resources/js/processes/scripts/components/ScriptEditor.vue:442]: Should commented code still be included?

File: resources/js/processes/scripts/components/ScriptEditor.vue [hunk 2]
```diff
@@ -481,6 +499,9 @@ export default {
   },
 
   methods: {
+    editorDidMount(editor) {
+      // const monaco = this.$refs.editor.monaco
```
Comment (TinCodes) [inline resources/js/processes/scripts/components/ScriptEditor.vue:503]: Empty method/commented code

**Review by TinCodes — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-5859060fdd.eng.processmaker.net

**Pull request merged**